pkg/dockerregistry/server/wrapped/zz_funcname_test.go

```diff
@@ -0,0 +1,68 @@
+package wrapped
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/docker/distribution/context"
+)
+
+func TestFuncName(t *testing.T) {
```

Comment:
would it make more sense to call this a wrapper test?

Reply:
renamed